mm2src/coins/qrc20.rs

```diff
@@ -1398,14 +1399,18 @@ impl MmCoin for Qrc20Coin {
                 .await?
         };
 
-        let sender_refund_fee = {
-            let sender_refund_output =
-                self.sender_refund_output(&self.swap_contract_address, swap_id, value, secret_hash, receiver_addr)?;
-            self.preimage_trade_fee_required_to_send_outputs(vec![sender_refund_output], &stage)
-                .await?
+        let total_fee = if include_refund_fee {
+            let sender_refund_fee = {
+                let sender_refund_output =
+                    self.sender_refund_output(&self.swap_contract_address, swap_id, value, secret_hash, receiver_addr)?;
+                self.preimage_trade_fee_required_to_send_outputs(vec![sender_refund_output], &stage)
+                    .await?
+            };
+            erc20_payment_fee + sender_refund_fee
+        } else {
+            erc20_payment_fee
```

Comment:
I think it is more readable if you move `erc20_payment_fee + sender_refund_fee` calculation from if else

```rust
        // Optionally calculate refund fee.
        let sender_refund_fee = if include_refund_fee {
            let sender_refund_output =
                self.sender_refund_output(&self.swap_contract_address, swap_id, value, secret_hash, receiver_addr)?;
            self.preimage_trade_fee_required_to_send_outputs(vec![sender_refund_output], &stage)
                .await?
        } else {
            BigDecimal::from(0) // No refund fee if not included.
        };

        let total_fee = erc20_payment_fee + sender_refund_fee;
 ```

Reply:
done in [789ec76](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051/commits/789ec7683331ef23b44441626440c220442d41d4)